File: app/src/main/java/br/com/danielsan/dstodolist/tasks/TasksFragment.java
package br.com.danielsan.dstodolist.tasks;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import br.com.danielsan.dstodolist.databinding.FragmentTasksBinding;

/**
 * Created by daniel on 25/03/16.
 */
public class TasksFragment extends Fragment {

    private FragmentTasksBinding binding;

    public static TasksFragment newInstance() {
        return new TasksFragment();
    }

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        binding = FragmentTasksBinding.inflate(inflater, container, false);
        return binding.getRoot();
    }

}

File: app/src/main/kotlin/br/com/danielsan/dstodolist/tasks/TasksFragment.kt
package br.com.danielsan.dstodolist.tasks

import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import br.com.danielsan.dstodolist.databinding.FragmentTasksBinding

/**
 * Created by daniel on 25/03/16.
 */
class TasksFragment : Fragment() {

    private var binding: FragmentTasksBinding? = null

    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        binding = FragmentTasksBinding.inflate(inflater, container, false)
        return binding!!.root
    }

    companion object {
        fun newInstance(): TasksFragment {
            return TasksFragment()
        }
    }

}
